### DISCUSSION TITLE
GQUIC analyzer

### DISCUSSION THREAD

**Discussion by petiepooo** [Security-Onion-Solutions/securityonion#6916] [Ideas] [answered]
More and more, we're seeing QUIC traffic.  It would be nice to integrate https://github.com/salesforce/GQUIC_Protocol_Analyzer into zeek in SecurityOnion.

**Comment by TOoSmOotH** [comment, marked as answer]
I'll look into it

**Comment by petiepooo** [comment]
There's also this one from Corelight that may be more current: https://github.com/corelight/zeek-quic

**Reply by dougburks** [reply]
I've added this note to https://github.com/Security-Onion-Solutions/securityonion/issues/6925.
